wh.order.import.cached.js

```diff
@@ -276,10 +278,53 @@ try {
                                       {concurrency: 1}
                                     )
                                       .then(function(){
-                                        console.log('cross-referenced and filled out lineitems against data from IronCache');
-                                        console.log('will send update(s) to loopback');
-                                        return client.models.ReportModel.updateRowsAsync(params.reportId, lineitems);
-                                      });
+                                        var connectionInfo = utils.loadOauthTokens();
```

Comment:
pulkitsinghal just now ShoppinPal member
you could blackbox this into a file ... like the logic is compressed earlier in:
var prepStockOrder = require('./jobs/cache-vend-products-for-stock-order.js');